src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortBamSpark.java

```diff
@@ -32,6 +34,11 @@
     protected int parallelism = 0;
 
     @Override
+    public CountingReadFilter makeReadFilter() {
+        return new CountingReadFilter("Allow all reads", ReadFilterLibrary.ALLOW_ALL_READS);
```

Comment:
I was just going to make the same comment. They're stateful, so I would
think not.

On Thu, Nov 19, 2015 at 2:15 PM, Louis Bergelson <notifications@github.com>
wrote:

> In
> src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortBamSpark.java
> <https://github.com/broadinstitute/gatk/pull/1159#discussion_r45386161>:
>
> > @@ -32,6 +34,11 @@
> >      protected int parallelism = 0;
> >
> >      @Override
> > +    public CountingReadFilter makeReadFilter() {
> > +        return new CountingReadFilter("Allow all reads", ReadFilterLibrary.ALLOW_ALL_READS);
>
> Does it make sense to use a counting read filter with spark tools?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/gatk/pull/1159/files#r45386161>.
>